tests/testsuite/doc.rs

```diff
@@ -2570,7 +2570,7 @@ fn doc_lib_false() {
                 doc = false
             "#,
         )
-        .file("bar/src/lib.rs", "")
+        .file("bar/src/lib.rs", "pub fn bar() {}")
```

Comment:
I'm not sure what this change is for?

Reply:
This must have been some left-overs for local testing that shouldn't have been left in, good catch. This is removed in d6ecc8521edd331924df58fa062d69de458ae06e .